support-frontend/assets/helpers/subscriptionsForms/formActions.ts

```diff
@@ -241,44 +239,23 @@ function setCsrCustomerData(
 		dispatch: SubscriptionsDispatch,
 		getState: () => CheckoutState,
 	): void => {
-		csrCustomerData.customer.email &&
-			dispatch(formActionCreators.setEmail(csrCustomerData.customer.email));
-		csrCustomerData.customer.email &&
-			dispatch(
-				formActionCreators.setConfirmEmail(csrCustomerData.customer.email),
-			);
-		csrCustomerData.customer.firstName &&
-			dispatch(
-				formActionCreators.setFirstName(csrCustomerData.customer.firstName),
-			);
+		const { email, firstName, country, street, city, postcode, state } =
+			csrCustomerData.customer;
+
+		email && dispatch(formActionCreators.setEmail(email));
+		email && dispatch(formActionCreators.setConfirmEmail(email));
+		firstName && dispatch(formActionCreators.setFirstName(firstName));
 		dispatch(formActionCreators.setLastName(csrCustomerData.customer.lastName));
 
 		dispatch(formActionCreators.setCsrUsername(csrUserName(csrCustomerData)));
 		dispatch(formActionCreators.setSalesforceCaseId(csrCustomerData.caseId));
 
 		const addressActions = addressActionCreatorsFor(addressType);
-		csrCustomerData.customer.country &&
-			addressActions.setCountry(csrCustomerData.customer.country)(dispatch);
-		csrCustomerData.customer.street &&
-			addressActions.setAddressLineOne(csrCustomerData.customer.street)(
-				dispatch,
-				getState,
-			);
-		csrCustomerData.customer.city &&
-			addressActions.setTownCity(csrCustomerData.customer.city)(
-				dispatch,
-				getState,
-			);
-		csrCustomerData.customer.postcode &&
-			addressActions.setPostcode(csrCustomerData.customer.postcode)(
-				dispatch,
-				getState,
-			);
-		csrCustomerData.customer.state &&
-			addressActions.setState(csrCustomerData.customer.state)(
-				dispatch,
-				getState,
-			);
+		country && addressActions.setCountry(country)(dispatch);
+		street && addressActions.setAddressLineOne(street)(dispatch, getState);
+		city && addressActions.setTownCity(city)(dispatch, getState);
+		postcode && addressActions.setPostcode(postcode)(dispatch, getState);
+		state && addressActions.setState(state)(dispatch, getState);
```

Comment:
This is so much easier to read 🤩 but we can also just pass these to `dispatch` rather than needing to call them again with `dispatch` and `getState`- Redux will do that for us internally:

```suggestion
		country && dispatch(addressActions.setCountry(country));
		street && dispatch(addressActions.setAddressLineOne(street));
		city && dispatch(addressActions.setTownCity(city));
		postcode && dispatch(addressActions.setPostcode(postcode));
		state && dispatch(addressActions.setState(state));
```